external/loaders/loaders/batches/_batch.py

```diff
@@ -112,6 +114,12 @@ def batches_from_mapper(
             add_wind_rotation_info(res),
         ]
 
+    if data_transforms is not None:
+        data_transform = dacite.from_dict(
+            vcm.ChainedDataTransform, {"config": data_transforms}
+        )
+        transforms.append(curry(data_transform.apply))
+
     transforms.append(add_derived_data(variable_names))
```

Comment:
Currently `add_derived_data` calls `vcm.DerivedMapping` on the data and selects the given variables. We could add this to the `vcm.ChainedDataTransform` to do it as part of that transformation. Need to decide whether to do it before or after all the other transforms. For now I left it after, since it does the variable selection.

Reply:
The challenge is that the inputs/outputs for the `vcm.DerivedMapping` are defined differently than the static definitions used for the registered data transforms.